src/api/libcellml/types.h

```diff
@@ -31,6 +33,13 @@ typedef std::shared_ptr<Units> UnitsPtr; /**< Type definition for shared units p
 class Variable; /**< Forward declaration of Variable class. */
 typedef std::shared_ptr<Variable> VariablePtr; /**< Type definition for shared variable pointer. */
 
+class XmlAttribute; /**< Forward declaration of XmlAttribute class. */
```

Comment:
Can I check that this is an internal header, since I don't think we want to expose these types in the API?